### ISSUE TITLE
Documentation of `utils.rs`

### ISSUE THREAD

**Issue by Razican** [SUPERAndroidAnalyzer/super#57]
Most of the code is still not documented. This is the tracking issue for `utils.rs` file documentation.
- [ ] Module level documentation.
- [x] `PARSER_CONFIG` constant.
- [x] `print_error()` function.
- [x] `print_warning()` function.
- [x] `print_vulnerability()` function.
- [x] `get_code()` function.
- [ ] `get_string()` function:
  - [x] Function level documentation.
  - [ ] Inline comments.
- [ ] Tests:
  - [ ] `it_get_code()` function.
  - [ ] `it_file_exists()` function.

**Comment by iamdharmveer** [comment]
I can do this for you 👍

**Comment by Razican** [comment]
@iamdharmveer wow! great, thanks!! :smiley:

**Comment by iamdharmveer** [comment]
Can you give me more details about the documentation 👍

**Comment by Razican** [comment]
For getters and setters, a small (even one line) explanation on the parameter they get and set would be enough. For constants, their function (in this case, `PARSER_CONFIG` is the parser configuration used for the XML parser).

Function descriptions should show what functions do, in this case, `print_error()` and `print_warning()` are used for errors and warnings of the analysis process itself, and they color them on stdout in different colors.

A good example of a documentation can be found in [this](http://fractal.global/utils-rs/fractal_utils/index.html) crate.

**Comment by iamdharmveer** [comment]
All changes in documentation.md Because I didn't found utils.rs so I'm asking about which file do you want to change ?

**Comment by iamdharmveer** [comment]
Ok thank you 👍

**Comment by Razican** [comment]
Nope, files are relative to the `src/` folder. So in this case, it would be the `src/utils.rs` file, and the documentation would need to be created using the [Rust documentation format](https://doc.rust-lang.org/book/documentation.html).

**Comment by iamdharmveer** [comment]
One more query if I want to join your organization how can it's happen 😄

**Comment by Razican** [comment]
Well, that is something that would happen with enough pull requests from you :) if you manage to get many pull requests merged in SUPER, you would probably become a collaborator, and after some time, a full member of the team :)

**Comment by iamdharmveer** [comment]
Thank You 😄  👍
